.github/workflows/ci.yml

```diff
@@ -0,0 +1,94 @@
+name: CI
+
+on: [pull_request, push]
+
+env:
+  ELIXIR_ASSERT_TIMEOUT: 2000
+  ELIXIRC_OPTS: "--warnings-as-errors"
+  ERLC_OPTS: "warnings_as_errors"
+  LANG: C.UTF-8
+
+jobs:
+  test:
+    name: Linux, ${{ matrix.otp_release }}, Ubuntu 16.04
+    continue-on-error: ${{ matrix.development }}
+    strategy:
+      matrix:
+        otp_release: ['OTP-23.0', 'OTP-22.3', 'OTP-22.0', 'OTP-21.3.8', 'OTP-21.0']
+        development: [false]
+        include:
+          - otp_release: master
+            development: true
+          - otp_release: maint
+            development: true
+    runs-on: ubuntu-16.04
+    steps:
+      - uses: actions/checkout@v1
+        with:
+          fetch-depth: 50
+      - name: Install Erlang/OTP
+        run: |
+          cd $RUNNER_TEMP
+          wget -O otp.tar.gz https://repo.hex.pm/builds/otp/ubuntu-14.04/${{ matrix.otp_release }}.tar.gz
+          mkdir -p otp
+          tar zxf otp.tar.gz -C otp --strip-components=1
+          otp/Install -minimal $(pwd)/otp
+          echo "::add-path::$(pwd)/otp/bin"
```

Comment:
Shall we cache this or something?

Reply:
We can but it's usually fast to fetch and extract.

Reply:
Ah, it is already precompiled, makes sense.